content/working_with/console/widgets/deployments.md

```diff
@@ -103,12 +83,47 @@ For detaching the current site, leave the `Site name` input empty and toggle the
 ![Set Site]( /images/ui/widgets/blueprint-deployments_set-site.png )
 
 
+#### Managing Labels
+
+1. Click **Manage Labels** in the action menu.
+2. Add/Remove labels to the deployment. 
```

Comment:
I'm not sure what's the best practice in such cases, but I think it would be better to use 'from' instead of 'to', as "Remove" is used after "Add":
```suggestion
2. Add/Remove labels from the deployment. 
```
or maybe:
```suggestion
2. Add/Remove labels to/from the deployment. 
```